src/api/app/views/webui/request/_actions_details.html.haml

```diff
@@ -15,11 +15,28 @@
           %li
             %hr.dropdown-divider
             = link_to((render partial: 'action_text', locals: { action: action_item }),
-                      request_show_path(number: bs_request.number,
-                                        request_action_id: action_item.id,
-                                        full_diff: diff_limit,
-                                        diff_to_superseded: diff_to_superseded_id),
-                      class: "dropdown-item #{action_item.id == active_action.id ? 'disabled' : ''}")
+                                                      request_show_path(number: bs_request.number,
+                                                                        request_action_id: action_item.id,
+                                                                        full_diff: diff_limit,
+                                                                        diff_to_superseded: diff_to_superseded_id),
+                                                      class: "dropdown-item #{action_item.id == active_action.id ? 'disabled' : ''}")
 %p
   = 'Showing' if submit_actions.count > 1
   %span.font-italic= request_action_header(action, bs_request.creator)
+
+- content_for :ready_function do
+  :plain
```

Comment:
Shouldn't we use `:javascript` here?

Reply:
Also, we could push this into a `.js` file and call it unobstrusively from our asset pipeline

Reply:
Talked offline about this and it's usage is too specific to benefit from extracting into a javascript file